util/measures/2019/pi-measures.csv

```diff
@@ -1,40 +1,40 @@
-measureId,title,description,isRequired,metricType,firstPerformanceYear,weight,objective,isBonus,reportingCategory,substitutes,measureSpecification,exclusion

```

Comment:
Why was this file changed?

Reply:
There was an update for 2019 data as well, mostly just affecting a few descriptions.